src/Components/Tabs/Tab.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
```

Comment:
```suggestion
import cn from 'classnames';
```
її назву зазвичай ось так скорочують